tests/taxRetentions.test.js

```diff
@@ -0,0 +1,158 @@
+import 'regenerator-runtime/runtime';
```

Comment:
```suggestion
import nock from 'nock'; 
import { APIMocker, newSession } from './fixtures';
import TaxRetentions from '../src/taxRetentions';
```

I tested swapping these lines here in other tests and the same error appears. This should do the fix